examples/integration_test.py

```diff
@@ -1323,6 +1431,10 @@ def resolve_envs(_conf):
         print('=' * 30, 'Verifying Admin API', '=' * 30)
         verify_admin()
 
+    if 'explicit-read' in modes:
```

Comment:
This does not need to be its own mode, call it from verify_avro()

Reply:
ok - c4ff376